### PR TITLE
Add authors

### PR DESCRIPTION
**Pull request by sylwiaszunejko**
@Ponewor and @moguchev please take a look if everything is correct.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -142,4 +142,6 @@ Dmitry Kropachev <dmitry.kropachev@gmail.com>
 Oliver Boyle <pleasedontspamme4321+gocql@gmail.com>
 Jackson Fleming <jackson.fleming@instaclustr.com>
 Sylwia Szunejko <sylwia.szunejko@scylladb.com>
-Karol Baryła <karol.baryla@scylladb.com>
\ No newline at end of file
+Karol Baryła <karol.baryla@scylladb.com>
+Marcin Mazurek <marcinek.mazurek@gmail.com>
+Moguchev Leonid Alekseevich <lmoguchev@ozon.ru>
```
Comment (wprzytula) [inline AUTHORS:147]: Last name mistakenly comes first here; IMO it should be:
`Leonid Alekseevich Moguchev`
Correct?

Comment (sylwiaszunejko) [reply AUTHORS:147]: @moguchev  gave me a thumbs up in the PR description, so I thought it was ok. This order was in the author of the commit.

Comment (sylwiaszunejko) [reply AUTHORS:147]: Now I am not sure

Comment (wprzytula) [reply AUTHORS:147]: I guess the order of names does not really make any difference here.

**Review by wprzytula — APPROVED** [review]

**Pull request merged**